### PR TITLE
Selfie: Add label for zoom level

### LINKED ISSUE #17: Add zoom information to zoom slider
It's not uncommon for the slider to not have a linear progress in some specific ratio ( 0.7x in Pixel 6 ) as they switch to another lens
As a first step we need to add the current ratio inside the slider, further in the process ( if possible ) add some know steps in the UI.

### PR DESCRIPTION
**Pull request by luk1337**
Change-Id: I2477d6ccfe44cdfdafef08a3beb449442dc4ff29

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -34,8 +34,10 @@ android {
 dependencies {
     implementation 'androidx.core:core-ktx:1.8.0'
     implementation 'androidx.appcompat:appcompat:1.4.2'
-    implementation 'com.google.android.material:material:1.6.1'
+    implementation 'com.google.android.material:material:1.7.0-alpha03'
```
Comment (luk1337) [inline app/build.gradle:37]: Needed for https://github.com/material-components/material-components-android/issues/2726

File: app/build.gradle [hunk 2]
```diff
@@ -34,8 +34,10 @@ android {
...
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel:2.5.1'
```
Comment (luk1337) [inline app/build.gradle:39]: Needed to fix dependency errors during build

**Comment by luk1337** [comment]
ADB Screenshot from 2022-08-10 15-08-12

File: app/build.gradle (hunk 1 above)
Comment (SebaUbuntu) [reply app/build.gradle:37]: Is this gonna be a problem when building with AOSP?

Comment (luk1337) [reply app/build.gradle:37]: ...if importing this exact version (or newer) won't be possible then yeah.

Comment (luca020400) [reply app/build.gradle:37]: I think the correct thing to do is importing all the libraries to match whatever gradle uses
That makes our lives much easier

**Review by luca020400 — APPROVED** [review]

**Comment by luk1337** [comment]
I think we should improve it in the future, it's kinda ugly as is.

**Pull request merged**